include/measurement_kit/internal/report/resubmit.h

```diff
@@ -0,0 +1,95 @@
+/* Part of Measurement Kit <https://measurement-kit.github.io/>.
+   Measurement Kit is free software under the BSD license. See AUTHORS
+   and LICENSE for more information on the copying conditions. */
+#ifndef MEASUREMENT_KIT_REPORT_RESUBMIT_H
+#define MEASUREMENT_KIT_REPORT_RESUBMIT_H
+
+/** @file measurement_kit/internal/report/resubmit.h
+ *
+ * @brief Internal FFI API to resubmit reports. Beware that internal APIs may
+ * change radically at any new release of Measurement Kit.
+ */
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif  /* __cplusplus */
+
+/** mk_report_resubmit_request_t is a request to resubmit a report. */
+typedef struct mk_report_resubmit_request mk_report_resubmit_request_t;
```

Comment:
For testability, it would make lots of sense to be able to specify a custom collector base URL. This would be helpful to direct the requests to a custom collector rather than to the default collector. OTOH, using the bouncer discovery mechanism is clearly conducive to a more thorough integration test in this case, because that is the code that will actually run in the real apps. This leads me to think that one perhaps may also want to set more variables here, like for example, the bouncer base URL. However, since this is a C API, there is no real issue here, because we can always _add_a new function to the API without breaking the API itself. Conclusion: there is no need to create a function for setting a custom collector URL now. This may be considered for later. I will write an integration test that runs a real test, without submitting the results, and then submit the results after the test.